docs/development.md

```diff
@@ -45,23 +48,23 @@ To gather the probe data that populates the probe API, run the following:
 ./manage.py import_probes
 ```
 
-The aggregations table depends on having separate tables for each version of
-a product and channel. The following command will ensure those tables are
-created based on the latest Firefox versions:
+The last step is to populate the aggregation tables with data from desktop
+Firefox.
 
-```
-./manage.py create_version_partitions
+```bash
+./manage.py import_aggs
```

Comment:
This should now be `./manage.py import_aggs <channel>` where channel is one of "nightly', "beta", or "release".